src/reducers/dataEntryErrorsReducer.js

```diff
@@ -3,6 +3,16 @@ import {
   UPDATE_SERVICE_ERROR,
   DISMISS_DATA_ENTRY_ERRORS,
 } from '../actions';
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
+  dsn: "https://f7a71e0f6282e1a79ba6f604080ec88e@o4507232810631168.ingest.us.sentry.io/4507235352313856",
```

Comment:
Nitpick: Values like this in this repo usually go in config.js.

Reply:
Fixed in 5559ebbb6adaaaa09ee40e7a70e02f1cc66b06f6